### ISSUE TITLE
post quant keras model with  UpSample2D ，after post-quantized  get a false output shape

### ISSUE THREAD

**Issue by sunzhe09** [tensorflow/tensorflow#43595]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):pip
- TensorFlow version (or github SHA if from source):2.3.0

**Command used to run the converter or code if you’re using the Python API**
If possible, please share a link to Colab/Jupyter/any notebook.

```
convert=tf.lite.TFLiteConvert.from_keras_model(model)
convert.target_ops=[tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
convert.optimizations=[tf.lite.Optimize.DEFAULT]
convert.representative
```

**The output from the converter invocation**

```
got a false tflite model,ps: the up  layer is 1x10x10x64 with an UpSample2D in the tflite i got a 1x1x1x64 output
```

**Also, please include a link to the saved model or GraphDef**

```
# Put link here or attach to the issue.
```

**Failure details**
If the conversion is successful, but the generated model is wrong,
state what is wrong:
- Producing wrong results and/or decrease in accuracy
- Producing correct results, but the model is slower than expected (model generated from old converter)

**RNN conversion support**
If converting TF RNN to TFLite fused RNN ops, please prefix [RNN] in the title.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by Saduf2019** [comment]
@sunzhe09 
Please share stand alone code to replicate issue faced along with error logs, or if possible share a colab gist with the issue reported.

**Comment by sunzhe09** [comment]
![image](https://user-images.githubusercontent.com/30410113/94382749-72dd4200-0170-11eb-9a6f-140c8302f291.png)
@Saduf2019  I don't get an error,when I check my quantized tflite model,I found the UpSample2D resized make a false output;Here is my model code:
![image](https://user-images.githubusercontent.com/30410113/94382848-d0718e80-0170-11eb-9d6b-1367d0bc727d.png)

**Comment by jvishnuvardhan** [comment]
@sunzhe09 Can you please share a simple standalone code to reproduce the issue? Did you check whether the model (before conversion) upsampling as you are expecting? Thanks!

**Comment by sunzhe09** [comment]
![image](https://user-images.githubusercontent.com/30410113/94504138-ae444300-023a-11eb-815e-6ff926881cc1.png)
@jvishnuvardhan I have check my training model summary,the Upsample2D is right shape;here is my  produce tflite code：
    
        f=open("./vallabels.txt", 'r')
        filenames=f.readlines()
   
        def representative_dataset_gen():
            for line in filenames:
                # Get sample input data as a numpy array in a method of your choosing.
                line=line.strip()
                if 'str' not in str(type(line)):
                    line = line.decode()
                image_path = line.split()[0]
            
                image=cv2.imread(image_path)
                image=cv2.resize(image,(320,320))
                image=cv2.cvtColor(image,cv2.COLOR_BGR2GRAY)
                image=np.expand_dims(image,axis=-1)
                image=np.expand_dims(image,axis=0).astype(np.float32)/255.
                print(image_path)
                yield [image]
        
        converter = tf.lite.TFLiteConverter.from_keras_model(model)
        converter.target_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
        converter.optimizations = [tf.lite.Optimize.DEFAULT]
        converter.representative_dataset = tf.lite.RepresentativeDataset(
            representative_dataset_gen) 
        converter.inference_input_type = tf.int8
        converter.inference_output_type = tf.int8
        tflite_model_quant = converter.convert()
        import pathlib
        tflite_model_quant_file = pathlib.Path("./")/"test.tflite"
        tflite_model_quant_file.write_bytes(tflite_model_quant)

**Comment by sunzhe09** [comment]
@Saduf2019 @jvishnuvardhan  any update？

**Comment by sunzhe09** [comment]
within new model-optimizer,the bug has been fixed;
